pkg/kubernetes/storageclass/v1alpha1/build.go

```diff
@@ -68,3 +68,13 @@ func (b *Builder) Build() (*storagev1.StorageClass, error) {
 	}
 	return b.sc.object, nil
 }
+
+// WithVolumeBind sets the volume binding mode of storageclass with provided argument.
+func (b *Builder) WithVolumeBind(bindingMode *storagev1.VolumeBindingMode) *Builder {
+	if bindingMode == nil {
```

Comment:
can we set error & return in case of nil arg?
If we think from caller's perspective, they need not invoke this build method if they dont have anything to set.

Reply:
Done